.phpcs.xml

```diff
@@ -22,7 +22,7 @@
 
 	<!-- Rules: Check PHP version compatibility -->
 	<!-- https://github.com/PHPCompatibility/PHPCompatibility#sniffing-your-code-for-compatibility-with-specific-php-versions -->
-	<config name="testVersion" value="5.6-"/>
+	<config name="testVersion" value="7.0-"/>
```

Comment:
WPGraphQL has officially deprecated PHP 5.6, despite PHP 5.6 support being broken atm. So it's probably best to leave it as `5.6` for now.